.circleci/config.yml

```diff
@@ -1,31 +1,20 @@
 version: 2
 
 jobs:
-  node10:
+  previous_node:
     docker:
-      - image: circleci/node:10.23 # Current BAS node version
+      - image: circleci/node:14 # Future BAS node version
```

Comment:
you dont use?

Reply:
No need to run on node 10 and 14. Not relevant anymore.